synapse/storage/data_stores/main/search.py

```diff
@@ -445,7 +445,9 @@ def search_msgs(self, room_ids, search_term, keys):
 
         results = list(filter(lambda row: row["room_id"] in room_ids, results))
 
-        events = yield self.get_events_as_list([r["event_id"] for r in results])
+        events = yield self.get_events_as_list(
+            [r["event_id"] for r in results], allow_redacted=False
+        )  # Don't return redacted
```

Comment:
pointless comment is a little pointless.

you could maybe explain *why* this is the right thing to do.